daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -401,6 +471,11 @@
               frequency of context switching, but increase the memory footprint.
             </xs:documentation>
           </xs:annotation>
+          <xs:simpleType>
```

Comment:
A defined simple type daf:OneOrGreater or some such would be more compact here as these would all remain one-liners with a type="daf:oneOrGreater" attribute.

Reply:
My only concern here, and why I went this route, is it makes the generator quite a bit more complicated. It doesn't currently have any logic to find other types or support restrictions not defined directly as a child of the tunable. It's still pretty dumb. I'd prefer that we just require this schema be verbose in favor a being easily parsable. There's plenty of ways one might want to refactor this file (e.g. imports/includes, unions), that I don't really want to have to support. I'd rather add comments in the schema file that specifiy that the generator expects a certain form. Thoughts?

Reply:
Adding a comment that the tunable generator needs dafext.xsd to be easily parseable (with some explanation what the certain form needs to be) is fine with me.

Reply:
I've added a new comment that describes the required format.